examples/cluster/mortgage-runner.py

```diff
@@ -0,0 +1,46 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+
+# pip install git+https://github.com/intel-go/ibis.git@develop
```

Comment:
Why is ibis needed in this setup?

Reply:
That's how `omniscripts` work - they require Ibis, as original intention was to have a repo with a set of scripts to compare Ibis and Pandas (and then with Modin).